zuul.d/adoption.yaml

```diff
@@ -13,6 +13,8 @@
           label: cloud-centos-9-stream-tripleo-vexxhost-xl
         - name: crc
           label: coreos-crc-extracted-xxl
```

Comment:
lets move these additions to a duplicate base job that we can merge for now and then when we are done with this work we can reconcile the two